### PR TITLE
[JBPM-9552] Disable testcontainers tests on windows and rhel8

### PR DESCRIPTION
**Pull request by gmunozfe**
**JIRA**: [JBPM-9552](https://issues.redhat.com/browse/JBPM-9552)

### CODE REVIEW

**Comment by gmunozfe** [comment]
@afalhambra could you review?

File: jbpm-workitem-itests/src/test/java/org/jbpm/workitem/springboot/samples/KafkaFixture.java [hunk 1]
```diff
@@ -77,6 +78,9 @@
     
     
     public void generalSetup() {
+        // Currently testcontainers are not supported out-of-the-box on Windows and RHEL8
+        assumeTrue(!System.getProperty("os.name").toLowerCase().contains("windows") 
+                && !System.getProperty("os.version").contains("el8"));
```
Comment (afalhambra) [inline jbpm-workitem-itests/src/test/java/org/jbpm/workitem/springboot/samples/KafkaFixture.java:83]: same comment regarding case sensitive, but not really required.

**Review by afalhambra — APPROVED** [review]
Looks good to me.
Nice job @gmunozfe

**Comment by gmunozfe** [comment]
@kiegroup/gatekeepers could you merge?

**Comment by ge0ffrey** [comment]
@gmunozfe Instead of pinging all 44 gatekeepers across KIE, would it be possible to only ping the jbpm gatekeepers for PR's like this?

**Comment by gmunozfe** [comment]
> @gmunozfe Instead of pinging all 44 gatekeepers across KIE, would it be possible to only ping the jbpm gatekeepers for PR's like this?

thanks for your advice @ge0ffrey, I didn't know there were 44 gatekeepers. Currently, there's no alias for jbpm gatekeepers (afaik) if someone can create it, it would be great.

**Review by MarianMacik — APPROVED** [review]

**Pull request merged**